meta/main.yml

```diff
@@ -29,7 +29,7 @@ dependencies:
   - role: brianshumate.consul
     when: patroni_dcs == "consul" and
           not patroni_dcs_exists
-  - role: retr0h.etcd
+  - role: andrewrothstein.etcd-cluster

```

Comment:
Thank you for suggestion of a new role for etcd. I need some time to test it out.